includes/wf_crm_webform_postprocess.inc

```diff
@@ -1973,22 +1973,25 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
    * @return string
    */
   public function getIpnRedirectUrl($type) {
-    $url = trim($this->node->webform['redirect_url']);
-    if ($url == '<none>' || $type == 'cancel') {
-      $url = url('node/' . $this->node->nid, array('absolute' => TRUE));
-    }
-    elseif ($url == '<confirmation>') {
-      $query = array('sid' => $this->submission->sid);
+    $confirmation_type = $this->node->getSetting('confirmation_type');
```

Comment:
Where is this node coming from? I've been testing Webforms not attached to nodes, which even if attaching Webforms to nodes, that's really just like an entity reference. Is this meant to be looking at setting on the Webform object?

Reply:
`node` == `webform`. I just never changed the property name before. Because in D7 the webform data was saved in the node.

Reply:
Ooookay. So, the property name is 'node' to avoid having to change so many things, but it's really a webform object. Got it.

We should make an issue to rename that later, perhaps before doing a proper release? It's gonna confuse people in the future.

Reply:
Thanks, Karin!